driver-core/src/main/com/mongodb/internal/connection/DefaultClusterFactory.java

```diff
@@ -52,6 +58,7 @@
  */
 @SuppressWarnings("deprecation")
 public final class DefaultClusterFactory {
+    private static final Logger LOGGER = Loggers.getLogger("DefaultClusterFactory");
```

Comment:
There is a specification for structured logging messages that ensures consistent logging within drivers. However, since those log messages do not adhere to any specification, the regular approach to logging is being used.

Reply:
Ack